egg-a-i-o.json

```diff
@@ -1,7 +1,7 @@
 {
     "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
     "meta": {
-        "version": "PTDL_v1"
+        "version": "PTDL_v2"
```

Comment:
What does this do?

Reply:
My question as well, I'm hoping it's just metadata, but I would need to see.

Reply:
v1 is exported older then panel version v 1.8 
v2 is exported after v1.8

you can import any v1 egg in to any panel version
v2 can only be imported in panel v 1.8 +

v2 is basically a newer version for eggs